src/app/beer_garden/api/entry_point.py

```diff
@@ -179,6 +181,9 @@ def _target_wrapper(
             # Also set up plugin logging
             beer_garden.log.load_plugin_log_config()
 
+            # Local plugin manager reference
+            lpm.lpm_proxy = lpm_proxy
+
```

Comment:
What is the benefit of passing forward lmp_proxy instead of connecting to it?

```python
from multiprocessing.managers import BaseManager
lpm.lpm_proxy=  BaseManager(address=('', 5000), authkey='abc')
lpm.lpm_proxy.connect()
```

Reply:
I couldn't get connecting to it working yet :upside_down_face: